Calendar/Get-RBASummary.ps1

```diff
@@ -338,33 +338,34 @@ function OutputRBADelegateSettings {
 }
 
 # RBA PostProcessing Steps
-function OutputRBAPostProcessing {
+function RBAPostProcessing {
     Write-Host -ForegroundColor Cyan "`r`n=====================";
     Write-Host -ForegroundColor Cyan "PostProcessing Steps";
     Write-Host -ForegroundColor Cyan "=====================`r`n";
     Write-Host -ForegroundColor Cyan "The RBA will format the meeting based on the following settings."
 
     #    Write-Host -ForegroundColor Cyan "`r`n`t RBA PostProcessing Steps";
     #    Write-Host -ForegroundColor Cyan "`t ------------------------------------   ---------------------------------";
-    Write-Host -ForegroundColor Cyan "`t AddOrganizerToSubject:                "$RbaSettings.AddOrganizerToSubject
-    Write-Host -ForegroundColor Cyan "`t DeleteSubject:                        "$RbaSettings.DeleteSubject
-    Write-Host -ForegroundColor Cyan "`t DeleteComments (Meeting body):        "$RbaSettings.DeleteComments
-    Write-Host -ForegroundColor Cyan "`t DeleteAttachments:                    "$RbaSettings.DeleteAttachments
-    Write-Host -ForegroundColor Cyan "`t RemovePrivateProperty:                "$RbaSettings.RemovePrivateProperty
-    Write-Host -ForegroundColor Cyan "`t DeleteNonCalendarItems:               "$RbaSettings.DeleteNonCalendarItems
-    Write-Host -ForegroundColor Cyan "`t RemoveForwardedMeetingNotifications:  "$RbaSettings.RemoveForwardedMeetingNotifications
-    Write-Host -ForegroundColor Cyan "`t RemoveCanceledMeetings:               "$RbaSettings.RemoveCanceledMeetings
-    Write-Host -ForegroundColor Cyan "`t EnableAutoRelease:                    "$RbaSettings.EnableAutoRelease
-    Write-Host -ForegroundColor Cyan "`t AddAdditionalResponse:                "$RbaSettings.AddAdditionalResponse
+    Write-Host -ForegroundColor Cyan @"
+    `t AddOrganizerToSubject:                $($RbaSettings.AddOrganizerToSubject)
+    `t DeleteSubject:                        $($RbaSettings.DeleteSubject)
+    `t DeleteComments (Meeting body):        $($RbaSettings.DeleteComments)
+    `t DeleteAttachments:                    $($RbaSettings.DeleteAttachments)
+    `t RemovePrivateProperty:                $($RbaSettings.RemovePrivateProperty)
+    `t DeleteNonCalendarItems:               $($RbaSettings.DeleteNonCalendarItems)
+    `t RemoveForwardedMeetingNotifications:  $($RbaSettings.RemoveForwardedMeetingNotifications)
+    `t RemoveCanceledMeetings:               $($RbaSettings.RemoveCanceledMeetings)
+    `t EnableAutoRelease:                    $($RbaSettings.EnableAutoRelease)
+    `t AddAdditionalResponse:                $($RbaSettings.AddAdditionalResponse)
+"@
+
```

Comment:
This empty line needs to be removed as it is against the code format policy.

Reply:
If you have any questions, make sure you run `.\.build\CodeFormatter.ps1` prior to pushing up changes.

Reply:
Sorry, I was rushing last night to get the comments resolved.